app/components/swimlanes/swimLanes.tsx

```diff
@@ -18,9 +18,26 @@ import { imageURL } from "../../utils/utils";
 import { DC_URL } from "../../config/constants";
 import useBreakpoints from "../../hooks/useBreakpoints";
 
-const SwimLanes = ({ lanesWithNumItems }) => {
+export type CollectionItem = {
```

Comment:
This sort of duplicates/overlaps with Alessandra's `CollectionCard` work from #160, so needs to be melded together